source/js/primary-nav.js

```diff
@@ -20,6 +20,20 @@ let primaryNav = {
       }
     }
 
+    const summaryBar = document.querySelector(".article-navbar");
+    if(summaryBar) {
+      const heroHeight = document.querySelector('.publication-hero-container').offsetHeight;
+      const navHeight = document.querySelector(".primary-nav-container").offsetHeight;
+      const total = heroHeight + navHeight - summaryBar.offsetHeight;
+      window.addEventListener("scroll", () => {
+        if(window.scrollY > total) {
+          summaryBar.style.marginTop = "0px";
+        } else{
+          summaryBar.style.marginTop = "-100px";
```

Comment:
Instead of putting hardcoded value, we can access custom CSS property's value using `.getPropertyValue(propertyName)`.

I can't recall the exact correct usage of it. I _think_ it's somewhere around

```js
window
      .getComputedStyle(summaryBar)
      .getPropertyValue(property);
```
😆 ?

Reply:
I learned a new thing! 

```javascript
getComputedStyle(element).getPropertyValue('--margin-top'); // for example 
```